versioned_docs/version-latest/farming-&-staking/farming/space-acres/space-acres-install.mdx

```diff
@@ -18,12 +18,20 @@ On Windows: download the [Microsoft Visual C++ Redistributable](https://learn.mi
 On Linux: follow [these instructions](https://learn.microsoft.com/en-us/cpp/linux/download-install-and-setup-the-linux-development-workload?view=msvc-170#linux-setup-ubuntu-on-wsl).
 :::
 
+:::note
+Please note that a wallet with a Substrate address is required to run the software.
+:::
+
 ## Step 1: Download the Space Acres Installer
 
 <DownloadButtons />
 
 ## Step 2: Install Space Acres
 
+:::caution
+Depending on your system, your screen may look different than the screenshots below.
```

Comment:
Can we get some macOS screenshots of the installation process and the Space Acres UI. Two different tabbed sections would be ideal.